koji_wrapper/validators.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+""" Collection of validator methods  """
+
+def validate_required(value):
```

Comment:
You could make this a static method, or pull it out of the class itself.

Reply:
I don't understand.  I have already pulled these out of the class, and they are reusable validator functions here, so I am unclear what you are suggesting

Reply:
I mis-read the diff. I thought it was part of a class in a previous file. Please disregard.